apps/chat/src/store/prompts/prompts.reducers.ts

```diff
@@ -249,12 +251,16 @@ export const promptsSlice = createSlice({
       state,
       { payload }: PayloadAction<{ folderId: string; name: string }>,
     ) => {
+      const parentId = getFolderFromId(
+        payload.folderId,
+        FolderType.Prompt,
+      ).folderId;
```

Comment:
getFolderIdFromEntityId
```suggestion
      const parentId = getFolderIdFromEntityId(payload.folderId);
```